cpp/test/stats/dispersion.cu

```diff
@@ -16,6 +16,7 @@
 
 #include "../test_utils.h"
 #include <gtest/gtest.h>
+#include <optional>
```

Comment:
`optional` requires C++17 -- which is fine with me : - ) -- but just want to make sure it's OK to use.